### PR TITLE
JBPM-8532 - update fireNewBranchEvent to include the original branch …

### PR DESCRIPTION
**Pull request by tsurdilo**
…name

Part of an ensemble:
* https://github.com/kiegroup/appformer/pull/712
* https://github.com/kiegroup/kie-wb-common/pull/2678

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-backend/src/test/java/org/kie/workbench/common/screens/impl/LibraryServiceImplTest.java [hunk 1]
```diff
@@ -766,6 +766,7 @@ public void addBranchTest() throws URISyntaxException {
 
         final NewBranchEvent newBranchEvent = newBranchEventArgumentCaptor.getValue();
         assertEquals("new-branch", newBranchEvent.getNewBranchName());
+        assertNotNull(newBranchEvent.getFromBranchName());
```
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-backend/src/test/java/org/kie/workbench/common/screens/impl/LibraryServiceImplTest.java]: could you please change it to `assertEquals("repo1-branch1", newBranchEvent.getFromBranchName());`?

Comment (tsurdilo) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-backend/src/test/java/org/kie/workbench/common/screens/impl/LibraryServiceImplTest.java]: updated - thank you

**Comment by paulovmr** [comment]
jenkins execute full downstream build

**Comment by tsurdilo** [comment]
@paulovmr full ds test fail not related to changes -- its in org.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest.testStandaloneEditorPerspective

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Comment by tsurdilo** [comment]
Jenkins execute full downstream build

**Pull request merged**